### PR TITLE
user-profile-module 

### PR DESCRIPTION
**Pull request by tanyushanel**

### CODE REVIEW

File: exadel-trainee-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -6,6 +6,10 @@ const routes: Routes = [
     path: 'login',
     loadChildren: () => import('./models/login/login.module').then((m) => m.LoginModule),
   },
+  {
+    path: 'user',
+    loadChildren: () => import('./models/user/user.module').then((m) => m.UserModule),
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/app-routing.module.ts:11]: I see @casterweb in his PR use **hr-profile** for naming everywhere. 
Could you please use **-profile** everywhere in your code
You can leave **path** as **user**, I think we will rename some pathes in future

**Comment by RomPavel** [comment]
Sorry If I didnt explain it well.
I mean if you have file/folder name with few words, please use 'my-file-name.ts' naming style
But in code use 'myFileName' style

File: exadel-trainee-app/src/app/models/user-profile/user-profile.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (Inunor) [inline exadel-trainee-app/src/app/models/user-profile/user-profile.component.spec.ts:1]: I think you can delete test files (or don't generate them at all, check angular ng generate flags for more info)

File: exadel-trainee-app/src/app/app.module.ts [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { UserProfileComponent } from './models/user-profile/user-profile.component';
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent, UserProfileComponent],
```
Comment (Inunor) [inline exadel-trainee-app/src/app/app.module.ts:9]: You shouldn't register your component here, you should move it to your module

**Review by Inunor — CHANGES_REQUESTED** [review]

**Review by RomPavel — APPROVED** [review]

**Review by Inunor — APPROVED** [review]

**Pull request merged**